test/cypress/integration/example.spec.js

```diff
@@ -0,0 +1,36 @@
+describe('Name of the group', () => {
+    beforeEach(() => {
+        // reset before tests
+        cy.viewport(1000, 500);
+    });
+
+    afterEach(()=>{
+        //
+    });
+
+    it('There should be a chart', () => {
+        cy.visit('/highcharts/3d/area');
```

Comment:
Question: Should we use the `before` hook for `visit()`? Or couple it under one `it()`? I'm referring this ["best pracite"](https://docs.cypress.io/guides/references/best-practices#Having-tests-rely-on-the-state-of-previous-tests)

Reply:
I somehow completely missed the `before` hook. It's probably cleaner to use those, and should make it easier to move tests around 👍 .